doc/src/modules/vector/api/classes.rst

```diff
@@ -5,10 +5,10 @@ Essential Classes in sympy.vector (doctrings)
 CoordSysCartesian
```

Comment:
Yes, because `CoordSysCartesian` class is replaced by `CartesianCoordinateSystem`. I will change it in the next commit.